voxel51/platform/api.py

```diff
@@ -78,6 +83,60 @@ def close(self):
         if self.keep_alive:
             self._requests.close()
 
+    def get_job_data_urls(self, task_config):
+        '''Retrieves signed URLs to download job input data.
+
+        Args:
+            task_config (voxel51.platform.task.TaskConfig): the task config
+
+        Returns:
+            a dictionary mapping input names to RemotePathConfig objects
+        '''
+        endpoint = self.base_url + "/jobs/" + task_config.job_id + "/url/data"
+        res = self._requests.get(endpoint, headers=self._header)
+        try:
+            _validate_response(res)
+            return {
+                k: voxu.RemotePathConfig(v)
+                for k, v in iteritems(_parse_json_response(res))
+            }
+        except (APIError, HTTPError) as e:
+            logger.warning(e)
+            return task_config.inputs
```

Comment:
Add an additional warning message saying that we're defaulting to the pre-populated inputs?